pkg/apis/sources/v1alpha1/cloudeventingress_types.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2019 The Knative Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/knative/pkg/apis/duck"
+	duckv1alpha1 "github.com/knative/pkg/apis/duck/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// Check that KubernetesEventSource can be validated and can be defaulted.
+var _ runtime.Object = (*CloudEventIngress)(nil)
+
+// Check that KubernetesEventSource implements the Conditions duck type.
+var _ = duck.VerifyType(&CloudEventIngress{}, &duckv1alpha1.Conditions{})
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// CloudEventIngress is the Schema for the cloudeventingresses API
```

Comment:
Add a period at the end to make this a complete sentence.

Reply:
Fixed. Thanks.